addons/account_edi/models/account_move.py

```diff
@@ -450,6 +454,34 @@ def _process_attachments_for_template_post(self, mail_template):
                 move_result.setdefault('attachments', []).extend(edi_attachments.get('attachments', []))
         return result
 
+    ####################################################
+    # Export Electronic Document
+    ####################################################
+
+    def action_download_electronic_invoice(self):
+        ids = ','.join(str(id) for id in self.filtered('edi_document_ids').ids)
+        if not ids:
+            return
+
+        return {
+            'type': 'ir.actions.act_url',
+            'url': '/web/edi_document/download?ids=' + ids,
```

Comment:
```suggestion
            'url': f'/web/edi_document/download?ids={ids}',
```
I usually prefer strings as I find them more readable, but it's up to you

Reply:
I like f string :+1:  too but if it's just for one variable at the end, I prefer string concatenation